examples/src/main/scala/caliban/http4s/ExampleApp.scala

```diff
@@ -56,15 +70,29 @@ object ExampleApp extends CatsApp with GenericSchema[Console with Clock] {
       maxDepth(30) @@                 // query analyzer that limit query depth
       timeout(3 seconds) @@           // wrapper that fails slow queries
       printSlowQueries(500 millis) @@ // wrapper that logs slow queries
-      apolloTracing                   // wrapper for https://github.com/apollographql/apollo-tracing
+      apolloTracing // wrapper for https://github.com/apollographql/apollo-tracing
+
+  def withErrorCodeExtensions[R](
```

Comment:
Great example! However I think it makes the "sample" use case a bit too complex. I also would like to keep the http4s, akka and finch examples identical. Instead of adding this everywhere, how about moving it to a unit test, or making a dedicated example here https://github.com/ghostdogpr/caliban/tree/master/examples/src/main/scala/caliban

Reply:
Removed the example code and created a little alinea in the docs for it. If that doesn't work I can move it to an example. Started with docs because creating an example for it felt quite a big thing to do straight away.